src/config/index.ts

```diff
@@ -74,6 +74,11 @@ export const getRpcServiceUrl = (): string => {
   return formatRpcServiceUrl(rpcUri, INFURA_TOKEN)
 }
 
+export const getPublicRpcUrl = (): string => {
+  const { publicRpcUri } = getChainInfo()
+  return formatRpcServiceUrl(publicRpcUri, INFURA_TOKEN)
```

Comment:
Is it intended that we share private Gnosis' token with user wallets? I don't think we should do that

Reply:
I doubt public RPCs need a token, do they?

Reply:
According to https://chainid.network/chains.json some of them do (rinkeby, ropsten) but I don't think we should use Safe's Infura token to do that because it has a rate limit and I'm not sure if we intend to share with users. I'd evaluate with the Infra team how we approach that. If we intend to pay for a private RPC endpoint for our users it should be a separate one at least.

Reply:
I know the whole point of this field was to not use a authed RPC. I initially used that formatter just for the sake of uniformity.
It would or would not add a token depending on the `authentication` param of the RPC.

Changed to bare `value` now.

Reply:
> Changed to a bare value now.

Thank you, with bare `value` it looks much less threatening

Reply:
Changed back to `formatRpcServiceUrl` but with an empty token :)

Reply:
it shouldn't require formatting if you ask me, though 🤷

Reply:
Just for consistency with other RPC getters + it abstracts the same value getter which otherwise would be duplicated.